contrib/test/integration/README.md

```diff
@@ -0,0 +1,19 @@
+# Fedora and RHEL Integration and End-to-End Tests
+
+This directory contains playbooks to set up for and run the integration and
+end-to-end tests for CRI-O on RHEL and Fedora hosts. Two entrypoints exist:
+
+ - `main.yml`: sets up the machine and runs tests
+ - `results.yml`: gathers test output to `/tmp/artifacts`
+
+When running `main.yml`, three tags are present:
+
+ - `setup`: run all tasks to set up the system for testing
+ - `e2e`: build CRI-O from source and run Kubernetes node E2Es
+ - `integration`: build CRI-O from source and run the local integration suite
+
+The playbooks assume the following things about your system:
+
+ - the `$GOPATH` is set and present for all shells (*e.g.* written in `/etc/environment`)
+ - CRI-O is checked out to the correct state at `${GOPATH}/src/github.com/kubernetes-incubator/cri-o`
```

Comment:
@cevich @mrunalp this chicken-and-egg situation is why having the job def deal with this sort of set-up is kind of weird. But I think we can live with it. I've re-thought some of the jobs I wrote to be more lean. Now there are basically four steps to all the jobs:

 - set up the user that our CI expects to use
 - install ansible and make sure ansible can work against `localhost`
 - create $GOPATH and clone CRI-O
 - run the `main.yml` playbook with the correct tags

Hopefully this will make it easier for y'all to have a self-sufficient playbook and make it clearer what the hidden dependencies are.